Doc/library/sqlite3.rst

```diff
@@ -1435,6 +1435,11 @@ Cursor objects
       and there is no open transaction,
       a transaction is implicitly opened before executing *sql*.
 
+      .. versionchanged:: 3.12
+
+         :exc:`DeprecationWarning` is raised if
+         :ref:`named placeholders <sqlite3-placeholders>` are used
+         and *parameters* is a sequence.
```

Comment:
Nitpicking. While it is technically correct description of the code behavior, I think it may be useful to mention not only what is wrong, but what would be correct. Something like "```is a sequence instead of a :class:`dict` ```" or "```is not a :class:`dict`, but a sequence```". What are your thoughts about this? Is it worth?

Reply:
Yes, I definitely think it is worth it. I am in favour of docs that communicate good practice first, and then deal with misuse.

Reply:
Come to think of it, I think I'm better off with improving the existing docs by better communicating best practice, before landing this PR.